static/src/stylesheets/module/atoms/_youtube.scss

```diff
@@ -158,4 +164,74 @@
             bottom: 8px;
         }
     }
+
+    .immersive-main-media__media & {
```

Comment:
hey, by this I meant that if the .youtube-media-atom has a .immersive-main-media__media as a parent. I think it's needed — or is there another, better way of doing this?